userd/settings.go

```diff
@@ -63,9 +63,9 @@ const settingsIntrospectionXML = `
 	</method>
 </interface>`
 
-// TODO: allow setting default-url-scheme-handler ?
 var settingsWhitelist = []string{
 	"default-web-browser",
+	"default-url-scheme-handler",
```

Comment:
This alone is not sufficient.  With this change, it would be possible to get userd to execute a call like:

    xdg-settings set default-url-scheme-handler foo.desktop

However, the man page says an additional argument is needed.  The example given is:

    xdg-settings set default-url-scheme-handler mailto evolution.desktop

So fixing this probably means a change to the D-Bus interface.

On the sandbox side, it looks like  the `xdg-settings` proxy script would need changes to handle the extra argument too: at the moment, it ignores all arguments after `$3`.